EKF/ekf.h

```diff
@@ -284,6 +284,9 @@ class Ekf : public EstimatorInterface
 	// set minimum continuous period without GPS fail required to mark a healthy GPS status
 	void set_min_required_gps_health_time(uint32_t time_us) { _min_gps_health_time_us = time_us; }
 
+	// set GPS sensor position
+	void set_GPS_pos_body(const Vector3f &gps_pos_body) { _params.gps_pos_body = gps_pos_body; }
```

Comment:
This is going to be overwritten when there's a parameter update. 

https://github.com/PX4/Firmware/blob/4f362f583511b6326fda859934905e553f74eb03/src/modules/ekf2/ekf2_main.cpp#L612-L614